src/adaptors/extra-finance/index.js

```diff
@@ -116,7 +116,7 @@ async function getPoolsData() {
 
     parsedFarmPoolsInfo.forEach(async (poolInfo) => {
       pools.push({
-        pool: `${poolInfo.pair}-${chain}`.toLowerCase(),
```

Comment:
pls revert this (as well as the change below). changing pool values requires manual updates on my end

Reply:
reverted. removed the duplicated pools in another way.